NthKey/View/Settings/Wallet/WalletEntityView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  WalletEntityView.swift
+//  WalletEntityView
+//
+//  Created by Sergey Vinogradov on 21.03.2021.
+//  Copyright © 2021 Purple Dunes. Distributed under the MIT software
+//  license, see the accompanying file LICENSE.md
+
+import SwiftUI
+import LibWally
+
+struct WalletEntityView: View {
+    let item: WalletEntity
+    let selected: Bool
+
+    var body: some View {
+        HStack {
+            Image(systemName: selected ? "checkmark.circle.fill" : "checkmark.circle")
+            Text("\(item.label ?? "N/A")")
+
+            Spacer()
+
+            Text("Thr.: \(item.threshold)")
```

Comment:
Let's rename the "Cosigners" section to "Wallet details", and then add a line: "Threshold: 1 of 2"  above the list of cosigners.

Reply:
fixed in 938f3e2ab5391df85c048c5eeb6987bc28f3c095